config.php

```diff
@@ -75,5 +77,70 @@
                 ],
             ],
         ],
+        'posts' => [
+            'path' => 'posts/{date|Y-m-d}/{-title}',
```

Comment:
Only to discuss about this.

If we create a folder to each date, we will need to create a index file inside each date folder to list the posts of this date or to redirect o other folder (i.e. /posts/). This is a problem because we will need to create another collection only to create this index file and will be more code to maintain.

Questions just to think and start the discussion:
- is it necessary to create this folder with the date?
- Could we use other pattern to path?
- What's the best suggestion to solve this problem?